__tests__/seed.test.js

```diff
@@ -183,5 +183,22 @@ describe('PATCH /api/articles/:article_id', () => {
                 expect(body.msg).toBe("bad request");
             });
     });
+});
 
+describe('GET /api/users', () => {
+    test('200: gets all users returns an array of objects', () => {
+        return request(app).get('/api/users')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.users.rows.length).toBe(4);
```

Comment:
This dot notation chain could be shorter if you look at your model and only send to the controller what you're looking for not the whole response. (This will make front end easier too)